bootstrap5-demo-with-translate/.naverc

```diff
@@ -0,0 +1 @@
+angular-slickgrid
```

Comment:
what is a `.naverc` file? I've never seen this kinda of file before and google didn't tell me much about it. Do we really need this file?

Reply:
i'll add it to `.gitignore`

Reply:
Removed in https://github.com/ghiscoding/angular-slickgrid-demos/pull/5/commits/5735c8167240d49b17b93d3128cb1c0637e77e60 and added to `.gitignore` in https://github.com/ghiscoding/angular-slickgrid-demos/pull/5/commits/14181c39cef77434ce50556d6dca53ab9feee981